### ISSUE TITLE
routing is not working in angularjs

### ISSUE THREAD

**Issue by karthiikramesh** [Microsoft/vscode#26864]
- VSCode Version:
- OS Version:

Steps to Reproduce:

1.<!DOCTYPE HTML>
<HTML>
    <HEAD>
    <link href="main.css" rel="stylesheet" type="text/css">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
   
    <TITLE>main</TITLE>
    <style>
      ul {
    list-style-type: none;
    margin: 0;
    padding: 0;
    overflow: hidden;
    background-color: #333;
    margin-bottom: 10px;
}

li {
    float: left;
}

li a {
    display: block;
    color: white;
    text-align: center;
    padding: 14px 16px;
    text-decoration: none;
}

li a:hover:not(.active) {
    background-color: #111;
}

.active {
    background-color: #4CAF50;
}   

div.gallery {
    border: 1px black #ccc;
    
}

div.gallery:hover {
    border: 2px snow #777;
}

div.gallery img {
    width: 400px;
    height: 300px;
}

div.desc {
    padding: 10px;
    text-align: center;
}

* {
    box-sizing: border-box;
}

.responsive {
    padding: 0 6px;
    float: left;
    width: 24.99999%;
    margin-top: 200px;
    margin-left: 130px;
}
@media only screen and (max-width: 700px){
    .responsive {
        width: 49.99999%;
        margin: 6px 0;
    }
}

@media only screen and (max-width: 500px){
    .responsive {
        width: 100%;
    }
}
    footer {
        text-align: center;
      background-color: #555;
      color: white;
      padding: 10px;
      margin-top: 900px;
    }

</style>
    </HEAD>
    <BODY ng-app="myApp"> 
     
   <ul>
  <li><a href="#home">Home</a></li>
  <li><a href="#offers">Offers</a></li>
  <li><a href="#contact">Contact</a></li>
  <li style="float:right"><a class="active" href="logout">Logout</a></li>
</ul>
  
  <div class="col-sm-6 "> 

<div class="responsive">
  <div class="gallery">
    <a target="_blank" href="#mens"><img ng-src="menw2.jpg" ></a>
    <div class="desc">Mens Watch</div>
  </div>
</div>
  </div>
  <div class="col-sm-6 "> 
<div class="responsive">
  <div class="gallery">
    <a target="_blank" href="#womens"><img ng-src="wmenw1.jpg" ></a>
    <div class="desc">Womens Watch</div>
  </div>
</div></div>
  

<div ng-view></div>

<footer class="container-fluid text-center">
  <p>@2017-all rights reserved</p>
</footer> 
<script>
  var app = angular.module("myApp", ["ngRoute"]);
app.config(function($routeProvider) {
    $routeProvider
    .when("/mens", {
        templateUrl : "mens.html",
        //  controller : "londonCtrl",
          template : "<h1>Banana</h1><p>Bananas contain around 75% water.</p>"
    })
    .when("/womens", {
        templateUrl : "/womens.html",
          template : "<h1>Banana</h1><p>Bananas contain around 75% water.</p>"
    })
      .when("/home", {
        templateUrl : "home.html"
    })
    .when("/contact", {
        templateUrl : "contact.html" 
    })
    .when("/logout", {
        templateUrl : "logout.html"  
    });
});
// app.controller("londonCtrl", function ($scope) {
//     $scope.msg = "I love London";
// });
</script>
    </BODY>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular-route.js"></script>
    <script src="main.js"></script>
</HTML> 

2.

**Comment by mjbvz** [comment]
Seems unrelated to vscode. Try stackoverflow
